### PR TITLE
add pypi!

### PR DESCRIPTION
**Pull request by ewdurbin**

### CODE REVIEW

File: libs/live-fns/pypi.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  switch (method) {
+    case 'v':
+      return {
+        subject: 'pypi.org',
+        status: 'v' + info.version,
+        color: info.version[0] === '0' ? 'orange' : 'blue'
+      }
+    default:
+      return {
+        subject: 'pypi',
```
Comment (olstenlarck) [inline libs/live-fns/pypi.js]: switch it to `pypi.org` too? for consistency? Or just make them both only `pypi`.

**Comment by amio** [comment]
Great!

File: libs/live-fns/pypi.js (hunk 1 above)
Comment (amio) [reply libs/live-fns/pypi.js]: +1 for both `pypi`

Like pip already did in pypi.org
https://pypi.org/project/pip/

File: libs/live-fns/pypi.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const axios = require('../axios.js')
+
+module.exports = async function (method, ...args) {
```
Comment (amio) [inline libs/live-fns/pypi.js]: Here we could use `package` instead of `...args` for better readability :D

File: libs/live-fns/pypi.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const endpoint = `https://pypi.org/pypi/${args[0]}/json`
+  const { info } = await axios.get(endpoint).then(res => res.data)
+
+  switch (method) {
+    case 'v':
+      return {
+        subject: 'pypi.org',
```
Comment (amio) [inline libs/live-fns/pypi.js]: How about drop `.org`?

I saw `pip` are using `pipy | v18.0` on pipy.org https://pypi.org/project/pip/

File: libs/live-fns/pypi.js (hunk 1 above)
Comment (ewdurbin) [reply libs/live-fns/pypi.js]: switched all to `pypi`

**Pull request merged**